docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1289,6 +1289,10 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
   +<Slider components={{ Thumb: CustomThumb }} />
   ```
 
+- CSS rework to match material design spec and make custom styles more intuitive.
+
+  ![image](https://user-images.githubusercontent.com/18292247/121712974-85cd5200-cb06-11eb-9776-f80ec10f48d1.png)
```

Comment:
The image doesn't display correctly on my screen: 1. subpixels issue 2. size fidelity issue. It feels strange/cheap.

Reply:
updated to

![image](https://user-images.githubusercontent.com/18292247/121830324-fa62e500-ccee-11eb-9aa5-7c802f497b36.png)

Reply:
I get the size fidelity issue with the updated version. I will push a change to share what I had in mind, then we can compare, see if it's an improvement or not.